crates/cxx-qt/include/cxxqt_connection.h

```diff
@@ -24,12 +23,15 @@ struct IsRelocatable<::QMetaObject::Connection> : ::std::true_type
 } // namespace rust
 
 namespace rust {
-namespace cxxqtlib1 {
+namespace cxxqtcommon1 {
```

Comment:
should we use `cxxqtcommon` and `cxx-qt-common` (directory) or just `cxxqt` and `cxx-qt` (directory) ?

Reply:
Hm, I think just `cxxqt`/`cxx-qt` is a bit nicer.
Doesn't really matter much though.

Reply:
Done as `cxxqt1`